compiler/lineinfos.nim

```diff
@@ -92,7 +92,7 @@ const
     warnResultShadowed: "Special variable 'result' is shadowed.",
     warnInconsistentSpacing: "Number of spaces around '$#' is not consistent",
     warnUser: "$1",
-    hintSuccess: "operation successful",
+    hintSuccess: "operation successful: $#",
```

Comment:
This changes the output of the other usages of ``hintSuccess``!

Reply:
which ones?
like I said here https://github.com/nim-lang/Nim/pull/8544#discussion_r207754392 I'm not seeing any actual use of `hintSuccess`:
```
nimgrep --ignoreStyle --ext:nim --recursive --oneline '\bhintSuccess\b'
/Users/timothee/git_clone/nim/Nim/tests/ccgbugs/twrong_setconstr.nim:116:     hintSuccess, hintSuccessX,
/Users/timothee/git_clone/nim/Nim/tests/sets/tsets.nim:175:     hintSuccess, hintSuccessX,
/Users/timothee/git_clone/nim/Nim/tests/sets/tsets.nim:188:   warnMax* = pred(hintSuccess)
/Users/timothee/git_clone/nim/Nim/tests/sets/tsets.nim:189:   hintMin* = hintSuccess
/Users/timothee/git_clone/nim/Nim/compiler/lineinfos.nim:40:     hintSuccess, hintSuccessX, hintCC,
/Users/timothee/git_clone/nim/Nim/compiler/lineinfos.nim:95:     hintSuccess: "operation successful",
/Users/timothee/git_clone/nim/Nim/compiler/lineinfos.nim:154:   warnMax* = pred(hintSuccess)
/Users/timothee/git_clone/nim/Nim/compiler/lineinfos.nim:155:   hintMin* = hintSuccess
```

Reply:
Oh ok, my bad.